daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,21 @@ enum RollOverState {
     None,
 }
 
+pub struct RollOverParams {
```

Comment:
I think these concern themselves with our "auto" roll-over feature, not roll-over per se.

cc @da-kami

```suggestion
pub struct AutoRollOverParams {
```